crowbar_framework/config/routes.rb

```diff
@@ -184,6 +184,11 @@
     end
   end
 
+  scope :sanity do
```

Comment:
As we are switching more an more to resources I would suggest `resource :sanity, only: [:show]`